assets/css/colors.scss

```diff
@@ -0,0 +1,10 @@
+$yellow: #FDC129;
+$light-turquoise: #EBF8F2;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#EBF8F2" to be "#ebf8f2" (color-hex-case)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5681062)